File: src_simple/python/aire_rectangle.py
#!/usr/bin/env python
# -*- coding=utf-8 -*-

""" aire_rectangle  -  11.03.2014 """

def aire_rectangle(largeur, longueur):
    """ Calcul de l'aire d'un rectangle """
    return largeur * longueur

if __name__ == "__main__":
    # lecture des variables
    largeur = float(raw_input("largeur = "))
    print("largeur = {0}".format(largeur))

    longueur = float(raw_input("largeur = "))
    print("longueur = {0}".format(longueur))

    # arriche le résultats
    print("Aire du rectangle = {0} ".format(aire_rectangle(largeur, longueur)))

File: src_simple/f90/aire_rectangle.f90
! aire_rectangle  -  11.03.2014

! ******************************************
! Calcul de l'aire d'un rectangle
! ******************************************

program aire_rectangle
    
    implicit none

    ! declaration
    real :: largeur, longueur

    ! Lecture des variables
    write(*,*) "Entrer la largeur"
    read(*,*) largeur
    write(*,*) "largeur = ", largeur

    write(*,*) "Entrer la longueur"
    read(*,*) longueur
    write(*,*) "longueur = ", longueur

    ! affiche le résultat
    write(*,*) "Aire du rectangle = ", largeur * longueur

end program aire_rectangle
